win32/gui.py

```diff
@@ -396,11 +396,16 @@ def latest_status(self):
             dogstatsd_status = DogstatsdStatus.load_latest_status()
             forwarder_status = ForwarderStatus.load_latest_status()
             collector_status = CollectorStatus.load_latest_status()
+            if sys.maxsize > 2**32:
+                python_architecture = "64bit"
+            else:
+                python_architecture = "32bit"
```

Comment:
Could you extract this logic in a method in `utils/platform.py`, and just call that method here (same thing in check_status)